keyRegistration/js/registration.js

```diff
@@ -75,14 +75,13 @@ $(document).ready(function()
         $('#options_self').addClass("hidden");
         $('#options_create').addClass("hidden");
         $('#options_general').addClass("hidden");
-    }
-
-    $('#self').click(function(){
+ 
```

Comment:
Line contains following spacing inconsistencies:
- Trailing whitespaces.

*SpaceConsistencyBear, severity NORMAL, section `default`.*

The issue can be fixed by applying the following patch:

```diff
--- a/keyRegistration/js/registration.js
+++ b/keyRegistration/js/registration.js
@@ -75,7 +75,7 @@
         $('#options_self').addClass("hidden");
         $('#options_create').addClass("hidden");
         $('#options_general').addClass("hidden");
- 
+
     $('#self').click(function () {
         self = $(this).prop("checked");
         self ? $('#email').addClass("hidden") : $('#email').removeClass("hidden");
```